python/data/lis/records/README.rst

```diff
@@ -0,0 +1,19 @@
+LIS Logical Records
+===================
+
+Test files for different Logical Record types
+
+=============== ==============================================================
+Filename        Desciption
+=============== ==============================================================
+inforec_01.lis  Structured record. Table is dense - I.e. all table entries are
+                recorded.
+inforec_02.lis  Structured record. Table is sparse - I.e. several table entries
+                are not recorded in record.
+inforec_03.lis  Unstructure record. I.e. not organised as a table
```

Comment:
We  need a valid 0-type-one-column-data test before that: it would also be a valid one-column table.